### PR TITLE
Fixes lightblue-platform/lightblue-core#376 : NPE when performing find on entity with hook

### PR DESCRIPTION
**Pull request by luan-cestari**
Depends on https://github.com/lightblue-platform/lightblue-audit-hook/pull/33

The issue https://github.com/lightblue-platform/lightblue-core/issues/376 have more details

### CODE REVIEW

File: crud/pom.xml [hunk 1]
```diff
@@ -90,6 +90,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.redhat.lightblue.hook</groupId>
+            <artifactId>lightblue-audit-hook</artifactId>
+            <scope>test</scope>
+        </dependency>
```
Comment (dcrissman) [inline crud/pom.xml]: Doesn't this create a circular dependency between core and audit-hook?

Comment (bserdar) [inline crud/pom.xml]: If that's  lightblue-rest/crud/pom.xml, it shouldn't.

Comment (dcrissman) [inline crud/pom.xml]: @bserdar you're right. I saw lightblue-core in the subject and it threw me. Disregard.

Comment (luan-cestari) [inline crud/pom.xml]: The build worked fine (as maven wouldn't allow circular dependency as far as I know)

**Pull request merged**